packages/vscode-extension/src/debugging/SourceMapsRegistry.ts

```diff
@@ -163,7 +163,7 @@ export class SourceMapsRegistry {
     if (this.sourceMapAliases) {
       // we return the first alias from the list
       for (const [alias, absoluteFilePath] of this.sourceMapAliases) {
-        if (absoluteFilePath.startsWith(absoluteFilePath)) {
+        if (sourceAbsoluteFilePath.startsWith(absoluteFilePath)) {
```

Comment:
is this a mistake? `sourceAbsoluteFilePath` is not defined

Reply:
It's the argument to `toSourceMapAliasedFilePath`, the thing we actually want to check in this function.
The previous implementation would check `x.startsWith(x)` and always return path relative to the first alias, which doesn't seem correct to me.